File: 01-FizzBuzz/java/src/test/java/FizzBuzzTest.java
import static org.junit.Assert.*;

import org.junit.Test;

public class FizzBuzzTest {

	@Test
    public void shouldReturn1for1() {
		FizzBuzz fizzBuzz = new FizzBuzz();
        String actualReturnValue = fizzBuzz.process(1);
        String expectedReturnValue = "1";
        assertEquals(expectedReturnValue, actualReturnValue);
    }
}

File: 01-FizzBuzz/csharp-dotnetframework/FizzBuzzTest/FizzBuzzTest.cs
using System;
using NUnit.Framework;

namespace Katas
{
    [TestFixture]
    public class NUnitTest1
    {
        [Test]
        public void ShouldReturn1For1()
        {
            FizzBuzz fizzBuzz = new FizzBuzz();
            string actualReturnValue = fizzBuzz.Process(1);
            string expectedReturnValue = "1";
            Assert.AreEqual(expectedReturnValue, actualReturnValue);
        }
    }
}